pypdf/constants.py

```diff
@@ -433,6 +433,31 @@ class InteractiveFormDictEntries:
     Q = "/Q"
     XFA = "/XFA"
 
+    # Common Field Flags
+    Ff_ReadOnly = 1
```

Comment:
This looks odd. Instead of using the existing class and just prefixing the values, we should probably use a new class and avoid the prefix by this to keep the API clean.

Reply:
oups I missed it below. already existing